File: InsurancePartnersApi/Models/Partner.cs
namespace InsurancePartnersApi.Models
{
    public class Partner
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Address { get; set; }
        public decimal PartnerNumber { get; set; }
        public string? CroatianPIN { get; set; }
        public int PartnerTypeId { get; set; }
        public DateTime CreatedAtUtc { get; set; }
        public string CreatedByUser { get; set; }
        public bool IsForeign { get; set; }
        public string ExternalCode { get; set; }
        public char Gender { get; set; }
    }
}

File: insurancepartnersfrontend/src/types/Partner.ts
export interface Partner {
    id: number,
    firstName: string,
    lastName: string,  
    partnerNumber: string;
    partnerTypeId: number,
    createdAtUtc: string,
    isForeign: boolean,
    gender: string,
    address?: string,
    createdByUser?: string,
    croatianPIN?: string,
    externalCode?: string
}
